src/SLCore.IntegrationTests/SLCoreTestProcessRunner.cs

```diff
@@ -51,7 +53,7 @@ public sealed class SLCoreTestProcessRunner : IDisposable
         bool enableVerboseLogs = false,
         bool enableAutoFlush = false)
     {
-        this.pathToBat = pathToBat;
+        launchParameters = new SLCoreLaunchParameters("cmd.exe", $"/c {pathToBat}");
```

Comment:
It seems some users of IntelliJ have their access to cmd.exe disabled by group policy we might need to think about that. For now it's OK but please create a ticket for this so we won't forget.

Reply:
As discussed offline, this is only in tests code. It will be fixed when for product code later